src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts

```diff
@@ -62,15 +62,18 @@ export class RegisterWorkflowModalComponent implements OnInit, AfterViewChecked,
   };
   public options = [
     {
-      label: 'Quickly register workflows from GitHub, Bitbucket, and GitLab',
+      label: 'Quickly register remote workflows',
+      extendedLabel: 'Toggle repositories from GitHub, Bitbucket, and GitLab to quickly create workflows on Dockstore.',
       value: 0
     },
     {
-      label: 'Create non-standard workflows from GitHub, Bitbucket, and GitLab',
+      label: 'Register custom remote workflows',
+      extendedLabel: 'Manually add individual workflows with custom metadata from repositories on GitHub, Bitbucket, and GitLab.',
```

Comment:
Same idea, also "values" are used above but "metadata" is used here. 
Maybe  `Manually add individual workflows at custom file paths` or `Manually add individual workflows at customized locations in repositories on GitHub, Bitbucket, and GitLab`

If using `locations` here, use it above. Same deal if using `file paths`.